todo_app/templates/index.html

```diff
@@ -10,9 +10,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+        {% for item in items %}
+          <div class="card" style="width: 18rem;">
+            <img src="..." class="card-img-top" alt="...">
```

Comment:
You'll need to provide a `src` for the image, e.g. `<img src="https://via.placeholder.com/300" class="card-img-top" alt="A placeholder image">`

Reply:
can you show me how to add an image?

Reply:
If you want to host the image as part of your app then you can make a `static` directory in `todo_app` and put it in there. Flask will serve files out of there by default.

Then you can access it with e.g. `http://localhost:5000/static/image.png` (where `image.png` is the name of the image file), and put that in the image `src` attribute